src/topic_partition_list.cpp

```diff
@@ -45,8 +45,7 @@ TopicPartitionsListPtr convert(const TopicPartitionList& topic_partitions) {
     TopicPartitionsListPtr handle(rd_kafka_topic_partition_list_new(topic_partitions.size()),
                                   &rd_kafka_topic_partition_list_destroy);
     for (const auto& item : topic_partitions) {
-        rd_kafka_topic_partition_t* new_item = nullptr;
-        new_item = rd_kafka_topic_partition_list_add(handle.get(),
+        rd_kafka_topic_partition_t* new_item = rd_kafka_topic_partition_list_add(handle.get(),
```

Comment:
I'm okay with making this change but only if the formatting looks right. You can make this one look like this, which would look better:

```c++
a* b = function_name(
    param1,
    param2,
    ...
);
```